Update Repair documentation

joaquincasares opened the pull request:
Minor changes made prior to sending this page to a customer.

rgooding commented on docs/operations/cassandra/repair.md:
```diff
@@ -1,63 +1,73 @@
-* Specify the number of tables to repair in parallel
+* Blacklist select tables.
+* Specify the number of tables to repair in parallel.
+* Specify the number of segments per VNode to repair.
```
we replaced the ability to specify the number of segments per vnode in Adaptive Repairs a while back, instead there is a "target segment size" field which checks the size of the table and splits it into an appropriate number of segments to hit the target size

rgooding commented on docs/operations/cassandra/repair.md:
```diff
@@ -1,63 +1,73 @@
 
-* Cron schedule repair: Same as **simple scheduled repair** but the schedule will be based on a Cron expression
+
+### Simple Scheduled Repairs
```
We no longer have "simple" schedules, it's only Immediate or Cron these days (but we have a fancy cron editor which makes them more friendly)

rgooding reviewed (changes requested):
Please see the comments I have left below

joaquincasares commented:
Thanks for calling that out! I updated the doc with new screenshots to match the new text.

I also changed the image locations to be with the repair documentation and made minor edits to match the new flow.

brainJamStark commented on docs/operations/cassandra/repair/index.md:
```diff
@@ -0,0 +1,70 @@
+* Exclude select tables.
+* Specify the number of tables to repair in parallel.
+* Specify the target segment size to repair.
+    * This value is used to group the target number of segments within a table into the same repair grouping.
```
I have updated this. please review @rgooding

joaquincasares commented:
@brainJamStark Thanks again for those image/lightbox edits!! It definitely looks better now. :muscle: 

I've updated the image tags to include accessibility-friendly tags that do not appear for visual browsers, but work well for screenreaders. These tags used to appear as tooltip tags a decade ago, but now render to the alt tag to avoid showing up in visual browsers:

<img width="920" height="46" alt="2025-08-25_17:16:45 49" src="https://github.com/user-attachments/assets/08ebf6b2-1990-4118-a13a-7d820e5c3fde" />
